lnd_test.go

```diff
@@ -2133,6 +2133,65 @@ func testNodeAnnouncement(net *networkHarness, t *harnessTest) {
 	}
 }
 
+func testNodeSignVerify(net *networkHarness, t *harnessTest) {
+	ctxb := context.Background()
+
+	// bob should be able to verify alice's signature
```

Comment:
In order to be consistent with the coding style of the project: all comments should be complete sentences and should also be flush with a coding fragment below it (meaning eliminate the extra new-line).